.github/workflows/ci.yml

```
@@ -1,75 +1,106 @@
   pull-requests: write
 
+env:
+  SKIP_MESSAGE: "[skip ci]"
+  NODE_VERSION: "20"
+
 jobs:
   tests:
+    name: Unit Tests
     runs-on: ubuntu-latest
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
```

actionlint (1.7.7)

property "skip_check" is not defined in object type {prereqs: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)